.github/mergeable.yml

```diff
@@ -0,0 +1,254 @@
+          - must_include:
+              regex: 'prio'
+              message: 'Please, add "prio" label'
+          - must_include:
+              regex: 'type'
+              message: 'Please, add "type" label'
+    fail:
+      - do: labels
+        add: 'blocked: mergeable'
+      - do: checks
+        status: 'failure'
+    pass:
+      - do: labels
+        delete: 'blocked: mergeable'
+      - do: checks
+        status: 'success'
+
+  # Remove label 'status: needs review' once pull request is approved.
+  # See: https://mergeable.readthedocs.io/en/latest/filters/payload.html
+  - when: 'pull_request_review.*'
+    name: 'Status check'
+    filter:
+      # If the pull request is approved by reviewer.
+      - do: payload
+        review:
+          state:
+            must_include:
+              regex: 'approved'
+    validate:
+      - do: label
+        must_include:
+          regex: 'status: needs review'
+    pass:
+      - do: labels
+        delete: 'status: needs review'
+      - do: checks
+        status: neutral
+
+  # Add label 'status: needs work' if reviewer requested changes.
+  - when: 'pull_request_review.*'
+    name: 'Status check'
+    filter:
+      # The author should have requested review.
+      - do: payload
+        review:
+          state:
+            must_include:
+              regex: 'changes_requested'
+    validate:
+      # The pull request should contain the needs review label.
+      - do: label
+        must_include:
+          regex: 'status: needs review'
+    pass:
+      # If it has the needs review label, remove it and add needs work.
+      - do: labels
+        add: 'status: needs work'
+        delete: 'status: needs review'
+
+  # Let's add "maintainers" team as reviewer once, 'optional: review requested' label is added.
+  - when: 'pull_request_review.*'
+    name: 'Request mainatainer for review'
+    filter:
+      # The pull request should have status request for review.
+      - do: payload
+        review:
+          state:
+            must_include:
+              regex: 'review_requested'
+    validate:
+      # If the pull request has the following label.
+      - do: label
+        begins_with:
+          match: 'optional: review requested'
+    pass:
+      # Request the maintainers for the review.
+      - do: request_review
+        teams: ['maintainers']
+
+  # Open Social always maintains multiple releases. We want to make sure that a proper milestone is added.
+  # This is only ignored if pull request is internal.
+  - when: pull_request.opened, pull_request.reopened, pull_request.milestoned, pull_request.demilestoned
+    name: "Valid milestone"
+    # See: https://mergeable.readthedocs.io/en/latest/operators/or.html
+    validate:
+      - do: or
+        validate:
+          # The pull request should have a milestone.
+          - do: milestone
+            no_empty:
+              enabled: true
+              message: 'Please select a milestone for this pull request.'
```

Comment:
I'm not sure if this is really helpful for contributors.
Imagine they open a PR, they don't know anything about what a milestone means to you. What version a bug might need to be backported to. 
This might lead to people selecting the wrong milestone, I'm not sure which is worse, no milestone or a wrong one. This might be better to leave out for now (?)